integration_tests/run.sh

```diff
@@ -1,37 +1,58 @@
 #!/bin/bash
 # Script for jenkins to run tests on polar2grid.
 
-set -exf
+set -ex
 export PATH="/usr/local/texlive/2019/bin/x86_64-linux":$PATH
 cd "$WORKSPACE"
 # Activate conda for bash.
 /data/users/davidh/miniconda3/bin/conda init bash
 # Restart the shell to enable conda.
 source ~/.bashrc
-conda env update -n jenkins_p2g_env -f build_environment.yml --prune
-conda activate jenkins_p2g_env
+conda env update -n jenkins_p2g_swbundle -f build_environment.yml
+conda activate jenkins_p2g_swbundle
 
-time=`date +"%Y%m%d-%H%M%S"`
-tarball_name=polar2grid-swbundle-$time
-./create_conda_software_bundle.sh $WORKSPACE/$tarball_name
-export POLAR2GRID_HOME="$WORKSPACE/$tarball_name"
+# Handle release vs test naming.
+prefix="$(cut -d'-' -f1 <<<"$GIT_TAG_NAME")"
+end="`date +"%Y%m%d-%H%M%S"`"
+# If the tag is correct and a version was specified, make a version release.
+if [[ "$GIT_TAG_NAME" =~ [pg]2g-v[0-9]+\.[0-9]+\.[0-9]* ]]; then
```

Comment:
I think you need a `.` (period) just before the last asterisk otherwise only numbers are allowed after the version number, right? I guess I'm wondering if `1.2.3a0` is allowed.

Reply:
p2g-v1.2.3a0 is allowed! the tag name "p2g-v1.2.3a0" returns "polar2grid-swbundle-v1.2.3a0".

Reply:
Ok so it is a partial pattern match. Then shouldn't the last `*` be a `+`? Right now it will accept `p2g-v1.2.` I think.